### DISCUSSION TITLE
Allow dragging the active tablet region to set it's position.

### DISCUSSION THREAD

**Discussion by mk56-spn** [ppy/osu#22462] [Ideas] [answered]
minor complaint but to me the use of sliders to set something on a coordinate system has always felt a little bit clunky.

It's a very niche issue so I'm mostly putting it out there tentatively to see what others think

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/discussions/21913

**Reply by mk56-spn** [reply]
my bad
